### PR TITLE
fix: instructions to install shim

### LINKED ISSUE #18: One of Install gvisor-containerd-shim steps is not correct.
When I follows the official doc[https://github.com/google/gvisor-containerd-shim/blob/master/docs/runtime-handler-quickstart.md] and try to install gvisor-containerd-shim, somthing goes wrong!
[root@localhost runsc]# LATEST_RELEASE=$(wget -qO - https://api.github.com/repos/google/gvisor-containerd-shim/releases | grep -oP '(?<="browser_download_url": ")https://[^"]*' | head -1)
[root@localhost runsc]# echo $LATEST_RELEASE
https://github.com/google/gvisor-containerd-shim/releases/download/v0.0.1/containerd-shim-runsc-v1.linux-amd64
[root@localhost runsc]# wget -O gvisor-containerd-shim
wget: missing URL
Usage: wget [OPTION]... [URL]...

Try `wget --help' for more options.

The step " wget -O gvisor-containerd-shim" should be "wget -O gvisor-containerd-shim $LATEST_RELEASE"
I have verified that "wget -O gvisor-containerd-shim $LATEST_RELEASE" is OK.

### PR DESCRIPTION
**Pull request by zhuangqh**
1. original `LATEST_RELEASE` is about `containerd-shim-runsc-v1`, we should choose the second line `gvisor-containerd-shim`. lol

2. should use `/etc/containerd/gvisor-containerd-shim.toml`  not yaml : )

 fixes #18

Signed-off-by: zhuangqh <zhuangqhc@gmail.com>

### CODE REVIEW

**Comment by zhuangqh** [comment]
> I signed it!

I signed it!

File: test/e2e/shim-install.sh [hunk 1]
```diff
@@ -7,8 +7,8 @@ set -ex
 # Build gvisor-containerd-shim
 if [ "${INSTALL_LATEST}" == "1" ]; then
 { # Step 1(release): Install gvisor-containerd-shim
-LATEST_RELEASE=$(wget -qO - https://api.github.com/repos/google/gvisor-containerd-shim/releases | grep -oP '(?<="browser_download_url": ")https://[^"]*' | head -1)
-wget -O gvisor-containerd-shim
+LATEST_RELEASE=$(wget -qO - https://api.github.com/repos/google/gvisor-containerd-shim/releases | grep -oP '(?<="browser_download_url": ")https://[^"]*' | awk 'NR==2')
```
Comment (ianlewis) [inline test/e2e/shim-install.sh]: I think this should be the following:
LATEST_RELEASE=$(wget -qO - https://api.github.com/repos/google/gvisor-containerd-shim/releases | grep -oP '(?<="browser_download_url": ")https://[^"]*gvisor-containerd-shim.linux-amd64' | head -1)

**Comment by ianlewis** [comment]
Also, please note that this fixes #18

File: test/e2e/shim-install.sh (hunk 1 above)
Comment (zhuangqh) [reply test/e2e/shim-install.sh]: @ianlewis done

**Pull request merged**